### PR TITLE
ci: upload and publish JUnit test results for conformance-multi-pool

### PR DESCRIPTION
**Pull request by mhofstetter**
This commit uploads and publishes JUnit test results of the connectivity tests in the conformance multi pool IPAM workflow.

### CODE REVIEW

File: .github/workflows/conformance-multi-pool.yaml [hunk 1]
```diff
@@ -195,3 +195,17 @@ jobs:
...
           path: cilium-sysdump-*.zip
           retention-days: 5
+
+      - name: Upload JUnits [junit]
+        if: ${{ always() }}
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+        with:
+          name: cilium-junits
+          path: cilium-junits/*.xml
+          retention-days: 5
```
Comment (viktor-kurchenko) [inline .github/workflows/conformance-multi-pool.yaml:205]: I was just wondering if 5 days are required. It looks like other actions use 2 days.

**Review by viktor-kurchenko — APPROVED** [review]
LGTM.

File: .github/workflows/conformance-multi-pool.yaml (hunk 1 above)
Comment (mhofstetter) [reply .github/workflows/conformance-multi-pool.yaml:205]: @viktor-kurchenko good point. see https://github.com/cilium/cilium/pull/27021 which is increasing it for the other workflows :)

Comment (viktor-kurchenko) [reply .github/workflows/conformance-multi-pool.yaml:205]: Nice

**Pull request merged**